fx/fx.go

```diff
@@ -16,7 +16,10 @@ type (
 
 	// FilterableFunc defines a func that should be passed to the Filter operator.
 	FilterableFunc func(interface{}) bool
-		
```

Comment:
Omit extra line here.

Reply:
I don't know why it shows like this.
There is extra line in source file.

Reply:
It just delete some white space and add an extra line here.
https://github.com/ReactiveX/RxGo/pull/40/files/1bba7e0e83de197091dfb0ece99e453aabacc356